pkg/volume/iscsi/iscsi_util_test.go

```diff
@@ -156,7 +156,7 @@ func TestWaitForPathToExist(t *testing.T) {
 		t.Errorf("waitForPathToExist: wrong code path called for %s", devicePath[1])
 	}
 
-	exist = waitForPathToExistInternal(&devicePath[1], 1, "fake_iface", os.Stat, fakeFilepathGlob2)
+	_ = waitForPathToExistInternal(&devicePath[1], 1, "fake_iface", os.Stat, fakeFilepathGlob2)
```

Comment:
This change is strange

Reply:
yes,may I remove this variable named exist?it as a reuturned bool variable does not appear below.accaully it is unused.

Reply:
From this context,this variable needs an error handling,right?

Reply:
I am not sure about this. 
This may be a mistake, the following judgment should be exist , refer to the above code.
Or this variable is redundant.